### PR TITLE
Remove unnecessary `sentry-sdk` dependencies causing problems in python27 builds

### PR DESCRIPTION
**Pull request by otwieracz**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

**Comment by otwieracz** [comment]
Applying same patch to `release-19.09` solved my issues from https://github.com/NixOS/nixpkgs/issues/71940 - Octoprint is working.

File: pkgs/development/python-modules/sentry-sdk/default.nix [hunk 1]
```diff
@@ -9,7 +9,7 @@ buildPythonPackage rec {
     sha256 = "f137cfb8bf709f69fa4634a7debd13284a3a590c374846285875be41d1fe87a8";
   };
 
-  checkInputs = [ django flask tornado sanic aiohttp bottle rq falcon pyramid celery ];
+  checkInputs = [ django flask tornado bottle rq falcon ];
```
Comment (jonringer) [inline pkgs/development/python-modules/sentry-sdk/default.nix]:
```suggestion
  checkInputs = [ django flask tornado bottle rq falcon ]
  ++ stdenv.lib.optionals isPy3k [ celery pyramid sanic aiohttp ];
```

```
      from pyramid.httpexceptions import HTTPException  # type: ignore
  ModuleNotFoundError: No module named 'pyramid'


  ----------------------------------------------------------------------
  Ran 4 tests in 0.001s

  FAILED (errors=4)
  Test failed: <unittest.runner.TextTestResult run=4 errors=4 failures=0>
  error: Test failed: <unittest.runner.TextTestResult run=4 errors=4 failures=0>
cannot build derivation '/nix/store/1wc8dp9hpb1vw6fq8m5ay3r5mzd3k8bz-env.drv': 2 dependencies couldn't be built
[11 built (2 failed), 25 copied (21.1 MiB), 5.3 MiB DL]
error: build of '/nix/store/1wc8dp9hpb1vw6fq8m5ay3r5mzd3k8bz-env.drv' failed
https://github.com/NixOS/nixpkgs/pull/72099
2 package failed to build:
python37Packages.sentry-sdk python38Packages.sentry-sdk

2 package were build:
octoprint python27Packages.sentry-sdk
```

File: pkgs/development/python-modules/sentry-sdk/default.nix [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{ stdenv, buildPythonPackage, fetchPypi, urllib3, certifi, django, flask, tornado, sanic, aiohttp, bottle, rq, falcon, pyramid, celery }:
+{ stdenv, buildPythonPackage, fetchPypi, urllib3, certifi, django, flask, tornado, bottle, rq, falcon }:
```
Comment (jonringer) [inline pkgs/development/python-modules/sentry-sdk/default.nix]: to fix your evaluation
```suggestion
{ stdenv, buildPythonPackage, fetchPypi, isPy3k, urllib3, certifi, django, flask, tornado, bottle, rq, falcon }:
```

File: pkgs/development/python-modules/sentry-sdk/default.nix [hunk 3]
```diff
@@ -1,4 +1,4 @@
-{ stdenv, buildPythonPackage, fetchPypi, urllib3, certifi, django, flask, tornado, sanic, aiohttp, bottle, rq, falcon, pyramid, celery }:
+{ stdenv, buildPythonPackage, fetchPypi, isPy3k, urllib3, certifi, django, flask, tornado, bottle, rq, falcon }:
```
Comment (jonringer) [inline pkgs/development/python-modules/sentry-sdk/default.nix]: need to add the other packages back, even if they aren't being used in py2 evaluation
```suggestion
{ stdenv, buildPythonPackage, fetchPypi, isPy3k, urllib3, certifi, django, flask, tornado, bottle, rq, falcon, celery pyramid, sanic, aiohttp }:
```

**Pull request merged**